### ISSUE TITLE
How to freeze generator during training?

### ISSUE THREAD

**Issue by the-entire-country-of-ireland** [NVlabs/stylegan2-ada-pytorch#145]
How do I freeze all the generator weights during training, while still training the discriminator?

I tried setting `do_Dmain, do_Gmain, do_Gpl, do_Dr1 = True, False, False, True` within `accumulate_gradients()` of `loss.py`, but this caused the generator to diverge.
https://github.com/NVlabs/stylegan2-ada-pytorch/blob/main/training/loss.py#L59

**Apart from `accumulate_gradients()` within `loss.py`, what else do I need to change to prevent the generator from continuing to train?**

These are the faces produced after resuming training from FFHQ for 40 epochs with the above flags set.
<img width="377" alt="image" src="https://user-images.githubusercontent.com/17838576/126219238-85b53e2b-63ae-499b-bb28-2813d943ca83.png">
